doc/denite.txt

```diff
@@ -1451,12 +1451,64 @@ abbr		(String)			(Optional)
 ==============================================================================
 CREATE KIND						*denite-create-kind*
 
-Note: Not implemented yet...
+Custom Kind can be created to contain extra actions for your custom source.
+Before create your new kind, you should have a look at existing kinds in
+rplugin/python3/denite/kind/*.py
+
+Instead of create new kind from sketch, you can extend one of the exiting
```

Comment:
To my ear, "from sketch" sounds weird. Did you mean "Instead of creating a new kind *from scratch"*?